### ISSUE TITLE
Let ASan unwind the stacks using CFI

### ISSUE THREAD

**Issue by ramosian-glider** [ramosian-glider/sanitizers#134]
Originally reported on Google Code with ID 133

```
Because this may potentially slow everything down, the user must have an option to separately
control the unwind algorithms for the error report and for malloc()/free()
```

Reported by `ramosian.glider` on 2012-12-10 14:17:30

**Comment by ramosian-glider** [comment]
Reported by `samsonov@google.com` on 2012-12-13 07:45:01
- **Status changed**: `Duplicate`
- **Merged into**: #137

**Comment by ramosian-glider** [comment]
```
Adding Project:AddressSanitizer as part of GitHub migration.
```

Reported by `ramosian.glider` on 2015-07-30 09:13:40
- **Labels added**: ProjectAddressSanitizer
